src/lib.rs

```diff
@@ -109,11 +113,33 @@ fn inner(markdown: &str, options: Options, h1_level: u8) -> Result<Vec<u8>, Stri
             Start(Tag::Item) => result.push(b'['),
             End(TagEnd::Item) => result.extend_from_slice(b"],"),
 
-            // TODO: Tables
-            Start(Tag::Table(_)) => todo!(),
-            End(TagEnd::Table) => todo!(),
-            Start(Tag::TableHead | Tag::TableRow | Tag::TableCell) => todo!(),
-            End(TagEnd::TableHead | TagEnd::TableRow | TagEnd::TableCell) => todo!(),
+            Start(Tag::Table(alignment_vector)) => {
+                let alignment_to_string = alignment_vector
+                    .iter()
+                    .map(|align| match align {
+                        Alignment::Left => "left",
+                        Alignment::Right => "right",
+                        Alignment::Center => "center",
+                        Alignment::None => "auto",
+                    })
+                    .collect::<Vec<&str>>();
+
+                result.extend_from_slice(b"#table(align: (");
```

Comment:
For consistency, I think we should remove the space here (as well as other unnecessary spaces, such as those after commas).